packages/router/src/__tests__/useMatch.test.tsx

```diff
@@ -55,7 +54,7 @@ describe('useMatch', () => {
       </LocationProvider>,
     )
 
-    expect(getByText(/Dunder Mifflin/)).toHaveStyle('color: green')
+    expect(getByText(/Dunder Mifflin/)).toHaveStyle('color: rgb(0, 128, 0)')
```

Comment:
Huh?! This was an unexpected change from going from jest to vitest

Reply:
Yeah. Not sure why it's different. I did a quick search but nothing immediately came up.

Reply:
Should we at least make the input match?

Reply:
I switched to using hex codes in [0575e22](https://github.com/redwoodjs/redwood/pull/10674/commits/0575e22c3f1f7df91b740f5cb5fd0be08000cdac)

Happy to revert if you don't like that. Or change in other ways